meshtastic/tests/test_mesh_interface.py

```
@@ -964,3 +966,264 @@ def test_timeago_fuzz(seconds: int) -> None:
+def test_concurrent_close_with_packet_id_generation() -> None:
+    """Test that close() properly handles concurrent packet ID generation."""
+    errors = []
+    stop_flag = threading.Event()
+
+    with MeshInterface(noProto=True) as iface:
+
+        def generate_ids() -> None:
+            try:
+                while not stop_flag.is_set():
+                    iface._generate_packet_id()
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)